app/constants/cfe_constants.rb

```diff
@@ -4,20 +4,25 @@ module CFEConstants
   DEFAULT_ASSESSMENT_VERSION = "6".freeze
   VALID_ASSESSMENT_VERSIONS = %w[6 7].freeze
 
-  # Valid CCMS Codes for proceeding types - probably need to get this from LFA in future
+  # Valid CCMS Codes for proceeding types - probably need to gPUBLIC_LAW_FAMILY_CCMS_CODESet this from LFA in future
```

Comment:
is this a copy and paste error?

Reply:
It looks like it doesn't it! 😆 
I'll resolve that too :+1: